jjb/dynamic/uperf.yml

```diff
@@ -18,6 +18,14 @@
         scp ${OPTIONS} -i ${PRIVATE_KEY} ${ORCHESTRATION_USER}@${ORCHESTRATION_HOST}:$HOME/.kube/config ${WORKSPACE}/kubeconfig
         export KUBECONFIG=${WORKSPACE}/kubeconfig
 
+        # check if cluster is up
+        date
+        oc get clusterversion
+        if [ $? == "1" ]; then
```

Comment:
Let's change this to if [[ $? != 0 ]]. Also think this logic should live in the workload, this way we can make use of it even when running it outside the pipeline.

Reply:
closing in favour of https://github.com/cloud-bulldozer/plow/pull/23